fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -407,6 +407,10 @@ public MvRefreshScheme copy() {
     @SerializedName(value = "maxMVRewriteStaleness")
     private int maxMVRewriteStaleness = 0;
 
+    @SerializedName(value = "partitionExprMaps")
+    private Map<GsonUtils.ExpressionSerializedObject, GsonUtils.ExpressionSerializedObject> serializedPartitionExprMaps;
+    private Map<Expr, SlotRef> partitionExprMaps;
```

Comment:
What if the MV is upgraded from a lower version and `partitionExprMaps ` is null?  can be compatible with old version?

Reply:
It will be generated from `partitionRefTableExprs` in `gsonPostProcess`